package.json

```diff
@@ -1,136 +1,132 @@
+        "test": "run-s test:format",
+        "test:ci": "run-s test release",
+        "test:format": "prettier-check '{{source,test}/**/*.js,webpack.config.js}'"
+    },
+    "lint-staged": {
+        "{{source,test}/**/*.js,webpack.config.js}": [
+            "prettier --write",
+            "git add"
+        ]
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/buttercup/buttercup-browser-extension.git"
+    },
+    "keywords": [
+        "buttercup",
+        "password",
+        "vault",
+        "login",
+        "secure"
+    ],
+    "author": "Perry Mitchell <perry@perrymitchell.net>",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/buttercup/buttercup-browser-extension/issues"
+    },
+    "homepage": "https://github.com/buttercup/buttercup-browser-extension#readme",
+    "devDependencies": {
+        "@babel/core": "^7.11.4",
+        "@babel/plugin-proposal-class-properties": "^7.10.4",
+        "@babel/plugin-proposal-function-bind": "^7.10.5",
+        "@babel/plugin-proposal-object-rest-spread": "^7.11.0",
+        "@babel/preset-env": "^7.11.0",
+        "@babel/preset-react": "^7.10.4",
+        "@blueprintjs/core": "^3.31.0",
+        "@blueprintjs/icons": "^3.20.1",
+        "@buttercup/channel-queue": "^0.5.0",
+        "@buttercup/config": "^0.1.0",
+        "@buttercup/dropbox-client": "^0.4.0",
+        "@buttercup/google-oauth2-client": "^0.2.1",
+        "@buttercup/googledrive-client": "^0.8.0",
+        "@buttercup/iconographer": "^1.2.0",
+        "@buttercup/locust": "^0.10.0",
+        "@buttercup/ui": "^3.1.2",
+        "babel-loader": "^8.1.0",
+        "babel-plugin-jsx-control-statements": "^4.1.0",
+        "browser-info": "^1.3.0",
+        "buttercup": "^4.13.1",
+        "classnames": "^2.2.6",
+        "color": "^3.1.2",
+        "concurrently": "^5.3.0",
+        "connected-react-router": "^6.8.0",
+        "console.style": "^0.2.2",
+        "copy-webpack-plugin": "^6.0.3",
+        "css-loader": "^4.2.1",
+        "debounce": "^1.2.0",
+        "dropbox": "^5.2.1",
+        "env-cmd": "^10.1.0",
+        "event-listener": "^1.1.0",
+        "eventemitter3": "^4.0.5",
+        "extract-domain": "^2.2.1",
+        "file-loader": "^6.0.0",
+        "history": "4.9.0",
+        "html-webpack-plugin": "^4.3.0",
+        "husky": "^4.2.5",
+        "lint-staged": "^10.2.11",
+        "marked": "^1.1.1",
+        "merge": "^1.2.1",
+        "mini-ttl": "^0.1.0",
+        "mkdirp": "^1.0.4",
+        "ms": "^2.1.2",
+        "mucus": "^1.0.0",
+        "node-noop": "^1.0.0",
+        "node-sass": "^4.14.1",
+        "npm-run-all": "^4.1.5",
+        "object-hash": "^2.0.3",
+        "path.join": "^1.0.0",
+        "pify": "^5.0.0",
+        "post-robot": "^10.0.39",
+        "prettier": "^2.0.5",
```

Comment:
Please keep prettier < v2, as v2 made some poor assumptions on formatting imo. They changed a lot, and a good portion of the community (myself included) don't agree with their changes (and they also require a lot of reformatting work). This might explain a lot of the changes in this PR -  reverting and reformatting would probably reduce the change size dramatically.